diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -118,24 +118,30 @@ namespace diff_drive_controller
      * \brief Wheel speed limiter setter
      * \param wheel_speed_limiter Wheel speed limiter
      */
-    void setWheelSpeedLimiter(WheelSpeedLimiter wheel_speed_limiter)
```

Comment:
This was in the header to let the compiler make it inline. Could you bring it back here, please?
In the future we could have a `diff_drive_controller.inl` file, or you can even add it now.